connector_search_engine/models/se_backend.py

```diff
@@ -28,38 +24,24 @@ class SeBackend(models.Model):
     specific_backend = fields.Reference(
         string='Specific backend',
         compute='_compute_specific_backend',
-        selection='_get_specific_backend_selection',
+        selection='_select_specific_model',
```

Comment:
@simahawk Are your sure? `selection='_select_specific_model'`

Reply:
```py
class Reference(Selection):
    type = 'reference'
```

Reply:
@simahawk My question is why `specific_model` and `specific_backend` use the same selection method? I miss something...

Reply:
@lmignon because they need to select the same set of models, the one registered as SE backends. Nothing else.

Reply:
@simahawk Initially, the idea behind specific_model and specific_backend was:
 -> specific_model provides the list of specialized models implementing an se.index. This field was used to allows each specialization to register itself  by using the method `register_spec_backend` https://github.com/akretion/connector-search-engine/blob/10.0/connector_algolia/models/se_backend_algolia.py#L31. (Your approach based on a naming convention replaces advantageously)
-> specific_backend must reference an **instance** of a specific_model. To build this list, we iterate on each existing specific model and search for instances... https://github.com/akretion/connector-search-engine/blob/10.0/connector_search_engine/models/se_backend.py#L44

We should therefore keep the `_get_specific_backend_selection`method